.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
     "cSpell.words": [
-        "yzhang"
-    ]
+    ],
+    "azure-pipelines.1ESPipelineTemplatesSchemaFile": true
```

Comment:
FYI not sure what that is, but VSCode recommended this to me. Is it plugin for intellisense on the 1ES templates?